platform/windows/display_server_windows.cpp

```diff
@@ -2187,8 +2195,41 @@ LRESULT DisplayServerWindows::MouseProc(int code, WPARAM wParam, LPARAM lParam)
 	return ::CallNextHookEx(mouse_monitor, code, wParam, lParam);
 }
 
-// Our default window procedure to handle processing of window-related system messages/events.
-// Also known as DefProc or DefWindowProc.
+// handle windows message received while CreateWindowEx is still on the stack and our data
+// structures are not fully initialized
```

Comment:
BTW I'm aware that some pre-existing comments may often have a different style. We're trying to enforce a consistent style for *new* comments at least so that progressively the surrounding code makes it obvious what the preferred style is. See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#comment-style-guide

Reply:
> Our comment style is to use normal sentence typography - start with a capital letter and end with a dot. Also I think it should be "window messages" and not the other way around?

we are both wrong :)    it should be "a single window message" which I have now put instead